### PR TITLE
publish machine timestamp

### PR DESCRIPTION
**Pull request by knorth55**
fix timestamp to publish now

### CODE REVIEW

File: src/manager/source_driver_ros1.hpp [hunk 1]
```diff
@@ -233,7 +233,7 @@ inline sensor_msgs::PointCloud2 SourceDriver::ToRosMsg(const LidarDecodedFrame<L
   }
   printf("frame:%d points:%u packet:%d start time:%lf end time:%lf\n",frame.frame_index, frame.points_num, frame.packet_num, frame.points[0].timestamp, frame.points[frame.points_num - 1].timestamp) ;
   // ros_msg.header.seq = s;
-  ros_msg.header.stamp = ros::Time().fromSec(frame.points[0].timestamp);
+  ros_msg.header.stamp = ros::Time::now();
```
Comment (garaemon) [inline src/manager/source_driver_ros1.hpp:239]: I guess the timestamp of `frame.points[0]` comes from the hardware and it is not stable or has some offsets. Is my understanding correct?
If so, could you add some comments about why we want to use `ros::Time::now()` here?

File: src/manager/source_driver_ros1.hpp [hunk 2]
```diff
@@ -247,7 +247,7 @@ inline hesai_ros_driver::UdpFrame SourceDriver::ToRosMsg(const UdpFrame_t& ros_m
     memcpy(&rawpacket.data[0], &ros_msg[i].buffer[0], ros_msg[i].packet_len);
     rs_msg.packets.push_back(rawpacket);
   }
-  rs_msg.header.stamp = ros::Time().fromSec(timestamp);
+  rs_msg.header.stamp = ros::Time::now();
```
Comment (garaemon) [inline src/manager/source_driver_ros1.hpp:256]: ditto

File: src/manager/source_driver_ros1.hpp (hunk 1 above)
Comment (knorth55) [reply src/manager/source_driver_ros1.hpp:239]: Yes.
frame.points[0] has offsets, it seem the duration from the process start?
I will add comments.

Comment (garaemon) [reply src/manager/source_driver_ros1.hpp:239]: > I will add comments.

Thanks 👍

Comment (knorth55) [reply src/manager/source_driver_ros1.hpp:239]: I added the comment.

File: src/manager/source_driver_ros1.hpp (hunk 2 above)
Comment (knorth55) [reply src/manager/source_driver_ros1.hpp:256]: I added a comment

**Review by garaemon — APPROVED** [review]
👍

**Pull request merged**